test.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+set -e
+npm run build
+echo "Stay tuned... tests are running!"
+
+./pkg/dist-node/index.bin.js clean
+
+./pkg/dist-node/index.bin.js optimize
+OPTIMIZED_SVG=./assets/dist/car.svg
+if [ -f "$OPTIMIZED_SVG" ]; then
+    echo "PASS (optimize): $OPTIMIZED_SVG exists."
```

Comment:
We could do a bit more if you want. You could pre-optimise some svgs and diff to ensure that we get roughly what we expect. Entirely optional.

Reply:
I created a test that compares the file size before and after the optimization. Hope that's fine. Comparing the content was a bit cumbersome

Reply:
sounds good!